WebKitBrowser/InjectedBundle/RequestHeaders.cpp

```diff
@@ -41,35 +43,8 @@ static PageHeaders s_pageHeaders;
 
 bool ParseHeaders(const string& json, Headers& out)
 {
-    struct HeadersData : public Core::JSON::Container
```

Comment:
This is here to avoid extra dependency on WebKitBrowser interfaces. Please keep it as it is.

Reply:
If the "interfaces/json/JsonData_WebKitBrowser.h" is defining the struct HeadersData : public Core::JSON::Container I do prefer the way it is. If the interface changes, compile time we will notice a problem building the Injectable bundle and I still think the compiler is my best friend :-) I would not like ot see the issue while I am testing in the field.
However, this statement is whiyhout nowing the full context. Lets discuss.